x-pack/plugins/enterprise_search/public/applications/app_search/index.tsx

```diff
@@ -16,13 +20,52 @@ export const AppSearch: React.FC = () => {
   const { enterpriseSearchUrl } = useContext(KibanaContext) as IKibanaContext;
 
   return (
-    <>
+    <Switch>
       <Route exact path="/">
-        {!enterpriseSearchUrl ? <Redirect to="/setup_guide" /> : <EngineOverview />}
+        {!enterpriseSearchUrl ? <Redirect to="/setup_guide" /> : <Redirect to="/engines" />}
       </Route>
-      <Route path="/setup_guide">
+      <Route exact path="/setup_guide">
         <SetupGuide />
       </Route>
-    </>
+      <Route>
+        <Layout navigation={<AppSearchNav />}>
+          <Switch>
+            <Route exact path="/engines">
+              <EngineOverview />
+            </Route>
+          </Switch>
+        </Layout>
+      </Route>
+    </Switch>
+  );
+};
+
+export const AppSearchNav: React.FC = () => {
+  const { enterpriseSearchUrl } = useContext(KibanaContext) as IKibanaContext;
+  const externalUrl = `${enterpriseSearchUrl}/as#`;
+
+  return (
+    <SideNav product={APP_SEARCH_PLUGIN}>
+      <SideNavLink to="/engines">
+        {i18n.translate('xpack.enterpriseSearch.appSearch.nav.engines', {
+          defaultMessage: 'Engines',
+        })}
+      </SideNavLink>
+      <SideNavLink isExternal to={`${externalUrl}/settings/account`}>
```

Comment:
Could use the same `legacyAppSearchUrl` recommended in prev comment

Reply:
Per above comment, will address this later in a routes constant refactor